crypto/keys/types.go

```diff
@@ -7,6 +7,9 @@ import (
 	"github.com/cosmos/cosmos-sdk/types"
 )
 
+// DefaultKeyDBName is the client's subdirectory where keys are stored.
```

Comment:
This doesn't seem to be used anywhere? If it is actually used but only by this package, may I suggest making it private?

Reply:
Agreed. Fixed